doc/init.md

```diff
@@ -33,24 +33,26 @@ If bitcoind is run with "-daemon" flag, and no rpcpassword is set, it will
 print a randomly generated suitable password to stderr.  You can also
```

Comment:
Not related to this PR, but this is no longer the case, right? It will generate a cookie instead.